addons/nvidia-bootstrap/image/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# Simple early detection of nvidia card
+grep 10de102d /proc/bus/pci/devices || exit 0
+
+# p2.xlarge
+# 00f0	10de102d	4b	        84000000	      100000000c	               0	        8200000c	               0	               0	               0	         1000000	       400000000	               0	         2000000	               0	               0	               0	nvidia
+
+
+# This is pretty annoying.... note this is installed onto the host
+chroot /rootfs apt-get update
+chroot /rootfs apt-get install --yes gcc
+
+mkdir -p /rootfs/tmp
+cd /rootfs/tmp
+# TODO: We can't download over SSL - presents an akamai cert
+wget http://us.download.nvidia.com/XFree86/Linux-x86_64/375.39/NVIDIA-Linux-x86_64-375.39.run
```

Comment:
You can add the following after this line to check that the sha3 hash matches:
`echo '5e5b9fbf12f4f926ed70c1fe39f71d9d9f154abea0268b1cf035982b34bd7c94baef7667e4f647cc19a62702b46f63b3c3df9f1589261f7138ed2ff151af63cc  NVIDIA-Linux-x86_64-375.39.run' | sha3sum -c -`